buildSrc/src/main/groovy/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -99,10 +103,14 @@ public void apply(Project project) {
         TaskProvider<Copy> copyUpgradeTask = configureCopyUpgradeTask(project, upgradeVersion, upgradeDir);
         TaskProvider<Copy> copyPluginsTask = configureCopyPluginsTask(project, pluginsDir);
 
+        final boolean dockerAvailable = isDockerAvailable();
```

Comment:
This code is part of the configuration of the `destructiveDistroTest` task though - I'm not expecting anyone to explicitly run the Docker tests during normal operations, it should just be part of the overall suite of distro tests. It's only tricky because we don't have Docker everywhere.

Reply:
you're right, thinking about it again, you're right, it's fine to have it here.